backends/qualcomm/builders/qnn_constants.py

```diff
@@ -266,6 +277,12 @@ class OpResizeNearestNeighbor:
     param_half_pixel_centers: str = "half_pixel_centers"
 
 
+@dataclass(init=False, frozen=True)
```

Comment:
Is it used for index_put?

Reply:
Yes, I choose Qnn ScatterND op to implement index put for llama use case. Because I have no idea to generate index_tensor with  Qnn ScatterElements op.